datahub-web-react/package.json

```diff
@@ -102,7 +102,7 @@
         "start:mock": "yarn run generate && BROWSER=none REACT_APP_MOCK=true craco start",
         "start:e2e": "REACT_APP_MOCK=cy BROWSER=none PORT=3010 craco start",
         "ec2-dev": "yarn run generate && CI=true;export CI;BROWSER=none craco start",
-        "build": "yarn run generate && CI=false REACT_APP_MOCK=false craco build && rm -rf dist/ && cp -r build/ dist/ && rm -r build/",
+        "build": "yarn run generate && CI=false REACT_APP_MOCK=false craco build && rm -rf dist/ && cp -r build/yarn/ dist/ && rm -r build/yarn/",
```

Comment:
curious - why do we want to place in build/yarn?

Reply:
It conflicts with the gradle `build` directory. This would delete build artifacts that gradle generates when it is making the assets jar. Moving it to inside the `build` directory allows javascript and gradle building to co-exist without conflict. It also handles the `clean` without being explicit.